src/main/java/hospital/management/Hospital/service/AppointmentService.java

```diff
@@ -37,6 +37,16 @@ public AppointmentDto getAppointmentById(Integer id){
         return AppointmentConverter.convertAppointmentToDto(appointmentRepository.findById(id).orElseThrow(NotFoundException::new));
     }
 
+    @Transactional
+    public AppointmentDto updateAppointmentStatus(Integer id, String status) {
+      Appointment appointment = appointmentRepository.findById(id).orElse(null);
+      if (appointment != null) {
+        appointment.setApproved(status);
+        appointmentRepository.save(appointment);
+      }
+      return getAppointmentById(id);
```

Comment:
I'm not sure what is the convention. Should we return the entity on update?

Reply:
if we don't need it we can just ignore the returned value i guess